src/main.cpp

```diff
@@ -1,9 +1,8 @@
-#include <iostream>
+#include "Dollar.h"
+ 
+int main(int argc, char** argv) {
 
-using namespace std;
-
-int main()
-{
-	cout << "Hello World!" << endl;
-	return 0;	
+	Dollar* dollar = new Dollar();	
```

Comment:
Oh and BTW, this a very common error in computer programming, it is called Memory Leak **https://en.wikipedia.org/wiki/Memory_leak**